source/refined-github.ts

```diff
@@ -198,6 +198,7 @@ import './features/linkify-full-profile-readme-title';
 import './features/same-page-definition-jump';
 import './features/new-repo-disable-projects-and-wikis';
 import './features/link-to-gihub-io';
+import './features/show-github-action-next-time';
```

Comment:
```suggestion
import './features/next-scheduled-github-action';
```

Every feature "Shows" or "adds" something, so it's implied